operator-apis/internal/handlers/handlers_test.go

```diff
@@ -0,0 +1,678 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/cloudfoundry-incubator/service-fabrik-broker/operator-apis/internal/config"
+
+	osbv1alpha1 "github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator/api/osb/v1alpha1"
+	"github.com/cloudfoundry-incubator/service-fabrik-broker/operator-apis/internal/constants"
+	"github.com/gorilla/mux"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Do we need both the types of clients? The client-go client and the one from controller run time?

Reply:
As discussed, `controller-runtime` dependency was added because `envtest` was used in UTs.